### DISCUSSION TITLE
Sentry Performance tracing does not work when lazy is set to true

### DISCUSSION THREAD

**Discussion by nieyuyao** [nuxt-community/sentry-module#592] [General]
### Version
@nuxtjs/sentry: 6.0.3
nuxt: 2.14.2

### Sentry configuration

```javascript
sentry: {
    tracesSampleRate: 0.1,
    lazy: true, //  works if set true
    vueOptions: {
      trackComponents: true,
    }
```

### What is actually happening?

Sentry browser tracing is broken if lazy is `false`. But it works if lazy is `true`. Is this to be expected or a bug?

**Comment by rchl** [comment]
When using `lazy`, the Sentry is loaded lazily (after the main app has loaded). So I would expect that it does affect performance tracing as it will miss some early events or won't be able to hook some things. But I'm not sure myself to which extent it affects tracing.

**Comment by nieyuyao** [comment]
OK. It maybe a bug in 6.x 😂.  Can I use version 7.x in nuxt@2 ?

**Comment by rchl** [comment]
This module uses Sentry 7.x - https://github.com/nuxt-community/sentry-module/blob/1ad3e0ab8ca777f3c52f60767d6c65c7613ba09e/package.json#L62-L66

**Reply by nieyuyao** [reply]
i mean is can i use @nuxtjs/sentry@7.x in nuxt@2 ? @rchl

**Reply by rchl** [reply]
Yes, this module is for Nuxt 2 (only).
